client/lib/signup/step-actions/index.js

```diff
@@ -428,7 +428,12 @@ export function createSiteAndAddDIFMToCart( callback, dependencies, step, reduxS
 				return;
 			}
 			siteSlug = result.siteSlug;
-			addDIFMLiteProductToCart( callback, { ...providedDependencies, siteSlug }, step, reduxStore );
+			addDIFMLiteProductToCart(
+				callback,
+				{ ...providedDependencies, siteSlug, siteId: result.siteId },
```

Comment:
In this context for 'new-site' is the site slug used at all?

Reply:
Yes, it is. The `siteSlug` is for the newly created site and is received as a result of the `createSiteWithCart` function. It is further used downstream in `addDIFMLiteProductToCart` -> `processItemCart`.